spec/components/geoblacklight/citation_component_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Geoblacklight::CitationComponent, type: :component do
```

Comment:
```suggestion
RSpec.describe Geoblacklight::CitationComponent, type: :component do
```

Don't depend on monkeypatching.